typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -943,6 +943,67 @@ export const polygonzkevm: ChainMetadata = {
   },
 };
 
+export const neutrontestnet: ChainMetadata = {
+  protocol: ProtocolType.CosmWasm,
+  domainId: 33333,
+  chainId: 33333,
+  name: Chains.neutrontestnet,
+  displayName: 'Neutron Testnet',
+  displayNameShort: 'Neutron Testnet',
+  nativeToken: {
+    name: 'Neutron',
+    symbol: 'NTRN',
+    decimals: 6,
+  },
+  blocks: {
+    confirmations: 1,
+    reorgPeriod: 0,
+    estimateBlockTime: 3,
+  },
+  rpcUrls: [{ http: 'http://54.149.31.83:26657' }],
+  isTestnet: true,
+  prefix: 'dual',
```

Comment:
again, this doesn't match the names used on our other branch. it's called `bech32prefix`. And we need a `slip44` field too

Reply:
sounds good, I'll rename prefix but will just wait for your other changes for the slip 44 part